### DISCUSSION TITLE
Many DNS queries

### DISCUSSION THREAD

**Discussion by aqwserf** [Security-Onion-Solutions/securityonion#2493] [Unsupported Versions] [answered]
Hi there,
after a fresh install of 2.3.21 on top of Ubuntu 18.04, I noticed many DNS requests made by SO (around 500/5mn):

![Screenshot from 2020-12-30 02-51-02](https://user-images.githubusercontent.com/29235861/103325228-26452380-4a4b-11eb-93cd-f4b357d010c0.png)

Source IP is SO and Dest IP is my OpenWRT. My OpenWRT responds correctly to any DNS/PTR requests made. DNS requests qre made for either:

- mirroring IP address (I have to use the tee module from OpenWRT to have WiFi traffic)
- localhost.MYSUBDOMAIN
- SO management IP   

Does anyone have an idea?

P.S: it didn't happen with 16.04

Thanks!

**Comment by Acewiza** [comment]
I saw the same thing and didn't find it remarkable.  Just alot more going on under the hood.  Lots of stuff uses DNS.

**Comment by aqwserf** [comment, marked as answer]
Yes, that's my guess too. I tried to kill every SO related processes/dockers, but requests are still made. I finally managed to stop them by editing `/etc/hosts`:
```
192.168.200.2	analyser.mirroring.MYDOMAIN
192.168.200.1	sensor.mirroring.MYDOMAIN
192.168.33.200	securityonion	securityonion.admin.MYDOMAIN
192.168.33.254	linksys.admin.MYDOMAIN
172.17.0.1	docker.localhost
```

Problem solved :)
